editor/editor_about.cpp

```diff
@@ -52,13 +54,23 @@ void EditorAbout::_notification(int p_what) {
 			_tpl_text->add_theme_constant_override("line_separation", 4 * EDSCALE);
 			_tpl_text->end_bulk_theme_override();
 
-			_license_text->begin_bulk_theme_override();
-			_license_text->add_theme_font_override("normal_font", font);
-			_license_text->add_theme_font_size_override("normal_font_size", font_size);
-			_license_text->add_theme_constant_override("line_separation", 4 * EDSCALE);
-			_license_text->end_bulk_theme_override();
+			license_text_label->begin_bulk_theme_override();
+			license_text_label->add_theme_font_override("normal_font", font);
+			license_text_label->add_theme_font_size_override("normal_font_size", font_size);
+			license_text_label->add_theme_constant_override("line_separation", 4 * EDSCALE);
+			license_text_label->end_bulk_theme_override();
 
 			_logo->set_texture(get_editor_theme_icon(SNAME("Logo")));
+
+			Ref<StyleBoxEmpty> empty_stylebox = memnew(StyleBoxEmpty);
```

Comment:
Unused.

Reply:
This is used at around line 135 by the way (if there was a global empty Stylebox to pick that would be welcome)

Reply:
No, line ~135 uses stylebox from line ~108 (in `_populate_list()`). This one was unused, but I see it's been removed.